lib/marked-renderer.js

```diff
@@ -1,7 +1,17 @@
 const { marked } = require('marked')
 
-/** A custom renderer that avoids wrapping `<img>` tags in `<p>` tags */
+/**
+ * Custom markdown renderer that avoids wrapping `<img>` image
+ * tags (or code with syntax highlighting) in `<p>` paragraphs
+ */
 class DesignSystemRenderer extends marked.Renderer {
+  code (code, language, isEscaped) {
+    return !language
+      ? super.html(code)
+      : super.code(code, language, isEscaped)
+        .replace('<code', '<code tabindex="0"')
```

Comment:
Ensure we add `tabindex="0"` to maintain keyboard focus + arrow key scrolling:

* https://github.com/alphagov/govuk-design-system/pull/1351